options/locale/locale_en-US.ini

```
@@ -141,6 +141,17 @@ confirm_delete_selected = Confirm to delete all selected items?
 name = Name
 value = Value
 
+filter = Filter
+filter.reset = Reset Filter
```

```suggestion
filter.clear = Clear Filter
```

Nit: This sounds a bit better to me.